### PR TITLE
Update headset-display.md

### PR DESCRIPTION
**Pull request by ErrorRaffyline0**
Added documentation for an issue where HMD displays show an incorrectly warped view, persistent across reinstalls of WMR.
Separated tracking-related jitter from performance-related jitter.
Added mentions about DisplayPort.
Minor fixes and clarity improvements.

### CODE REVIEW

**Comment by PRMerger4** [comment]
@TheJokingTeen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by TheJokingTeen** [comment]
Sorry for the large amount of commits. The two most recent commits are only very minor wording changes.

**Comment by ErrorRaffyline0** [comment]
@MaxWang-MS Please consider reviewing this, @qianw211 seems to be inactive.

File: enthusiast-guide/headset-display.md [hunk 1]
```diff
@@ -98,12 +98,12 @@ If you see "HoloLens Sensors" with a yellow exclamation mark in Device Manager,
 
 ## The images shown on the headset's displays appear warped or incorrectly scaled.
 
-This issue is likely caused by your headset's or graphics cards drivers or Windows software experiencing problems with adjusting the displayed content to appear correctly on your headset. You can try the following to fix the issue:
+This issue is likely caused by your headset's or graphics card's drivers or Windows software experiencing problems with adjusting the displayed content to appear correctly on your headset. You can try the following to fix the issue:
 
-* Disable automatic upscaling in Mixed Reality.
+* Disable automatic upscaling in mixed reality.
```
Comment (ErrorRaffyline0) [inline enthusiast-guide/headset-display.md:103]: Why the removal of capitalisation?

**Comment by ErrorRaffyline0** [comment]
@vtieto you were right about the capitalization (although there are some sentences there there is more capitalization to be removed (at least on the main branch)). The PR above also fixes some misleading info.

**Pull request merged**